tornado/test/twisted_test.py

```diff
@@ -67,6 +67,7 @@
     import _thread as thread
 else:
     import thread
+    ResourceWarning = None
```

Comment:
this is probably the second-most-interesting change (because python2 does not have ResourceWarning)

(this comment moved here from `runtests.py` - that was better handled with a "# noqa", but this case is more tricky)